### ISSUE TITLE
508-defect-2 [COGNITION] Videos should have captions and a link to transcript of

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#21020]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Noah

## Details

Videos should have appropriate captions embedded in the video (that could be turned on or off. Youtube has this feature). There should be a link following video to a transcript of the video for people who are unable to view the video

## Acceptance Criteria

- [ ] Confirm captions in video are accurate
- [ ] And link to transcript of video
- [ ] Transcript page should include header matching content to the title of video

**Comment by laflannery** [comment]
There is no enough information to complete this task - we are also not responsible for videos, the YouTube team is.  Therefore, I will be closing this in 1 week unless told otherwise.
